recipes/noqt/meta.yaml

```diff
@@ -47,8 +36,10 @@ outputs:
       noarch: generic
     requirements:
       run_constrained:
+        # While qt-main still doesn't point to qt6, it may in the future...
+        - qt-main <6.0a0|>=7.0a0
```

Comment:
If you allow `qt7` here, then `noqt` needs a separate run-constraint (probably a better idea anyway than depending on the subpackages IMO), because just depending on `noqt{5,6}` would then still allow `noqt` to be installed with `qt7`.

Reply:
```suggestion
        # While qt-main still doesn't point to qt6, it may in the future...
        - qt-main <6.0a0
```